data_processors/pipeline/orchestration/tumor_normal_step.py

```diff
@@ -19,6 +19,7 @@
     _mint_libraries
 from data_processors.pipeline.services import workflow_srv, metadata_srv, fastq_srv
 from data_processors.pipeline.tools import liborca
+from data_processors.pipeline.tools.liborca import _handle_rerun
```

Comment:
Could put this under `__init__.py` of `data_processors.pipeline.orchestration` package, pls. Along with other underscore methods there.

```
data_processors.pipeline.orchestration._handle_rerun(...)
```

Reply:
Resolved in 7f4f03c68c8146c4034f141bf454a5ff7b4f5321